tools/cldr-code/src/main/java/org/unicode/cldr/util/XMLSource.java

```
@@ -590,6 +590,7 @@ public String getSourceLocaleIdExtended(
                 list.add(
                         new LocaleInheritanceInfo(
                                 locale, path, LocaleInheritanceInfo.Reason.value));
+                // Since we’re not resolving, there’s no way to look for a Bailey value here.
```

We should throw an exception if we try for a bailey value with a non-resolving source.

---

Good point. and i'll add an exception up in `getPathsWhereFound()`, certainly.  This code, however is two levels deeper, in the XMLSource override.

---

ok. Are you going to update the test as part of this PR?

---

Yes, I'll probably work on that Monday.

---

Test found some bugs !

---

throws exception, passes tests